lib/pymedphys/_data/zenodo.py

```diff
@@ -28,7 +28,7 @@ def get_all_zenodo_file_data(record_name):
 
 @functools.lru_cache()
 def get_all_zenodo_file_data_by_id(record_id):
-    response = requests.get(f"{BASE_URL}{record_id}")
+    response = requests.get(f"{BASE_URL}{record_id}", timeout=30)
```

Comment:
Can all data be downloaded in this time span? What's the need for this change to be included with the poetry upgrade PR?

Reply:
This is a timeout to wait for an initial response for the server. The reason I've put this in is that pylint now warns if the timeout value isn't set.